media-video/pipewire/pipewire-9999.ebuild

```diff
@@ -97,6 +104,29 @@ PATCHES=(
 # with changes as necessary.
 limitsdfile=40-${PN}.conf
 
+# Copied with modifications from media-sound/pulseaudio-9999 from b.g.o (has sign-offs)
+_meson_native_feature() {
+	echo "-D${2:-${1}}=$(multilib_native_usex ${1} enabled disabled)"
+}
+
+# Copied with modifications from sys-apps/systemd/systemd-9999
+meson_multilib() {
```

Comment:
Should be converted into `meson_use <USE flag> [option name]` style function.

Reply:
I believe the current meson_native_feature and meson_native_enabled should do that. Please let me know if that's not it.